easybuild/easyconfigs/m/MACS2/MACS2-2.2.5-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'PythonPackage'
+
+name = 'MACS2'
+version = '2.2.5'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/taoliu/MACS/'
+description = "Model Based Analysis for ChIP-Seq data"
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/taoliu/MACS/archive']
```

Comment:
Any reason for now downloading from PyPI like the existing `MACS` easyconfigs do?

```python
source_urls = [PYPI_SOURCE]
sources = [SOURCE_TAR_GZ]

Reply:
according to the changelog on latest release for 2.2.5

https://github.com/taoliu/MACS/releases

They are including only on the github release some extra things for testing purposes.

But if these tests are not being triggered it's true we could stay with the PyPI one.

Reply:
Any update on this one?